### PR TITLE
web: Adapt the storage overview to stop using ProposalResult

### PR DESCRIPTION
**Pull request by ancorgs**
## Problem

As a first step to start using `AgamaProposal` instead of `MinGuidedProposal` at the web UI, we disabled the storage summary at the overview page (commenting out the usage of the component `Overview/StorageSection`.

## Solution

This enables the component back, but now reading from the correct source of information.

## Screenshots

The "storage" entry is back home. See following screenshot.

storage_section

### CODE REVIEW

File: web/src/components/overview/StorageSection.tsx [hunk 1]
```diff
@@ -104,81 +37,71 @@ const Content = ({ children }) => (
...
- *
- * @param {object} props
- * @param {Proposal} props.proposal
  */
 export default function StorageSection() {
-  const availableDevices = useAvailableDevices();
-  const result = useProposalResult();
-
-  if (result === undefined) return;
+  const drives = useConfigDevices().filter((d) => d.name);
```
Comment (joseivanlopez) [inline web/src/components/overview/StorageSection.tsx:42]: NP: all drives should have a name.

Comment (joseivanlopez) [reply web/src/components/overview/StorageSection.tsx:42]: BTW,  at this moment `useConfigDevices` returns drives only, but the model will be extended with LVM, etc. Should we filter drives here to be sure everything is a drive?

Comment (ancorgs) [reply web/src/components/overview/StorageSection.tsx:42]: Yes. This only adapts the summary to the very same cases that are handled by the storage section. We are not working on LVM support yet. If LVM is used, both this part and the storage configuration page would be equally broken.

We could of course filter, just to be safe. But then the information would be wrong (well, not fully precise) when we start adding LVM. I think I prefer to add a TODO to make it obvious and let it explode so it's not overlooked.

File: web/src/components/overview/StorageSection.tsx [hunk 2]
```diff
@@ -37,9 +37,13 @@ const Content = ({ children }) => (
 
 /**
  * Text explaining the storage proposal
+ *
+ * TODO: The current implementation asumes there are only drives and no other kind of devices like
```
Comment (joseivanlopez) [inline web/src/components/overview/StorageSection.tsx:41]: Typo: assumes.

**Review by joseivanlopez — APPROVED** [review]

**Pull request merged**